### ISSUE TITLE
Dark mode applied to any web sites not only the browser like Samsung Internet's Dark mode?

### ISSUE THREAD

**Issue by 3oheicrw** [minbrowser/min#1903]
What does Appearance/Enable Site theme does?

Turn it on or off doesn't have any effects. I used to think it's about forcing sites to use Dark theme by Min but it turned out to be wrong. The sites always use their own Light theme regardless of the Dark theme my Min is using.

It should be something like Samsung Internet's Dark mode, which applied to the web sites too a global Dark theme.

Min doesn't have extensions support. If it has, apart from Adblock (which is built-in), Dark Reader extension is always needed. If Min will not have extensions please add a built-in feature like what Dark Reader does. Thanks.

**Comment by addyett** [comment]
I'm having the same issue, enabling/disabling dark mode has no effect on any site other that duckduckgo searches

**Comment by 3oheicrw** [comment]
No one even care?

**Comment by djtech-dev** [comment]
No, you need to use the [UserScript](https://github.com/PalmerAL/min-dark-mode-userscript) for that.
